variables-skyline/index420.html

```diff
@@ -82,7 +82,7 @@
       .bb2b {
         width: 100%;
         height: 100%;
-        background-image:
+        background:
```

Comment:
I tried to shorten the test description for this lesson: "You can see the hard color change at the top of the `bb2b` section. Change the `bb2b` gradient type from `linear-gradient` to `repeating-linear-gradient`. This will make the gradient repeat throughout the rest of the element, giving you some stripes and saving you from having to add a bunch of elements to create them."